mangum/adapter.py

```diff
@@ -57,8 +57,9 @@ def handler(self, event: dict, context: dict) -> dict:
 
     def handle_http(self, event: dict, context: dict) -> dict:
         server, client = get_server_and_client(event)
-        headers = [
-            [k.lower().encode(), v.encode()] for k, v in event["headers"].items()
+        headers = event.get("headers", dict())
```

Comment:
I'd prefer to use the dict literal syntax here, e.g. `event.get("headers", {})`

Reply:
changes pushed.